src/coreclr/System.Private.CoreLib/src/System/Reflection/MdImport.cs

```diff
@@ -171,38 +171,44 @@ public static bool IsTokenOfType(int token, params MetadataTokenType[] types)
         public override string ToString() => string.Create(CultureInfo.InvariantCulture, stackalloc char[64], $"0x{Value:x8}");
     }
 
-    internal unsafe struct MetadataEnumResult
+    internal ref struct MetadataEnumResult
```

Comment:
Why make it a `ref struct`?

Reply:
This isn't really needed. I changed this when I was investigating how best to convert the `_Enum` call and wanted to confirm it wasn't being used in a managed array. There is no reason it needs to be a `ref struct` at this point, but it captures the intent better so I am inclined to leave it.